aspnetcore/blazor/components/layouts.md

```diff
@@ -367,6 +367,61 @@ The following rendered HTML markup is produced by the preceding nested layout. E
 
 When routable components are integrated into a Razor Pages app, the app's shared layout can be used with the components. For more information, see <xref:blazor/components/prerendering-and-integration>.
 
+:::moniker range=">= aspnetcore-8.0"
+
+## Layout sections
+
+To control the content in layouts from Razor components, Blazor supports *sections* using the following built-in components:
+
+* `SectionOutlet`: Renders content provided by `SectionContent` components with matching `SectionId` or `SectionName` arguments. A `SectionOutlet` component can render the content of several `SectionContent` components.
+
+* `SectionContent`: Provides content as a <xref:Microsoft.AspNetCore.Components.RenderFragment> to `SectionOutlet` components with a matching `SectionId` or `SectionName`. Only one `SectionOutlet` component can render the content of a `SectionContent` instance.
+
+In the following example, the `MainLayout` component implements an increment counter button for the app's `Counter` component.
+
+Add a `TopbarSection` static `object` to the `MainLayout` component (`Shared/MainLayout.razor`) in an `@code` block:
```

Comment:
Better to start with the example using `SectionName` as it will be the most used scenario. And then describe how the same thing can be achieved with `SectionId` and explain that this could be useful for creating a library. For more details see Updated Usage Examples https://github.com/dotnet/aspnetcore/issues/46937#issuecomment-1470667355

Reply:
This bit won't be a quick fix. I'll work on this Friday morning and ping you back.

Reply:
Sorry I didn't review the PR earlier. I will be looking out for the notification for the review request.